### ISSUE TITLE
how many steps should i wait till it starts speaking ?

### ISSUE THREAD

**Issue by manmaynakhashi** [OlaWod/FreeVC#36]

**Comment by OlaWod** [comment]
what does `starts speaking` mean?
according to configs/freevc.json, it logs every log_interval=200, steps, evals every eval_interval=10000 steps.

**Comment by manmaynakhashi** [comment]
my model is not giving any output after 200k steps , how many steps do i need to wait till i get some output ?

**Comment by manmaynakhashi** [comment]
i have trained my model for 100k steps after your pretrained checkpoint and it's still giving me noise as a output.

**Comment by OlaWod** [comment]
Do you mean, using your 100k checkpoint, you get noise as output no matter what you input?
In that case I think maybe your training process has some problem. Common reasons may be unsuitable learning rate, batch size, dataset size, problematic data preprocessing, etc.

**Comment by manmaynakhashi** [comment]
i added vctk dataset along with my dataset and output is coming now, thank you for sharing this great work .
